src/main/java/operation/BalanceOperationHandler.java

```diff
@@ -0,0 +1,9 @@
+package operation;
+
+import service.BalanceService;
+
+public class BalanceOperationHandler implements OperationHandler {
+    public void changeBalance(BalanceService balanceService, String item, Integer quantity) {
+        balanceService.updateBalance(item, quantity);
+    }
+}
```

Comment:
does this handler work the same as ReturnHandler and SupplyHandler?
then we don't actually need 3 same handlers